azext_iot/sdk/digitaltwins/controlplane/models/azure_data_explorer_connection_properties_py3.py

```diff
@@ -77,8 +82,8 @@ class AzureDataExplorerConnectionProperties(TimeSeriesDatabaseConnectionProperti
         'event_hub_consumer_group': {'key': 'eventHubConsumerGroup', 'type': 'str'},
     }
 
-    def __init__(self, *, adx_resource_id: str, adx_endpoint_uri: str, adx_database_name: str, event_hub_endpoint_uri: str, event_hub_entity_path: str, event_hub_namespace_resource_id: str, adx_table_name: str=None, event_hub_consumer_group: str=None, **kwargs) -> None:
-        super(AzureDataExplorerConnectionProperties, self).__init__(**kwargs)
+    def __init__(self, *, adx_resource_id: str, adx_endpoint_uri: str, adx_database_name: str, event_hub_endpoint_uri: str, event_hub_entity_path: str, event_hub_namespace_resource_id: str, identity=None, adx_table_name: str="AdtPropertyEvents", event_hub_consumer_group: str="$Default", **kwargs) -> None:
+        super(AzureDataExplorerConnectionProperties, self).__init__(identity=identity, **kwargs)
```

Comment:
Are we missing integration to this new identity property for the ADT ADX connection?

Reply:
when I asked, I was told that we should not worry about UAI for data history - this will be a feature that we would implement later. Will double check again.